pkg/controller/knativeserving/openshift/openshift.go

```diff
@@ -581,3 +565,77 @@ func installNetworkPolicies(instance *servingv1alpha1.KnativeServing) error {
 	}
 	return nil
 }
+
+func exposeMetricToServiceMonitor(instance *servingv1alpha1.KnativeServing) error {
+	// getOperatorNamespace return namespace where knative-serving-operator has been installed
+	namespace, err := getOperatorNamespace()
+	if err != nil {
+		log.Info("no namespace defined, skipping ServiceMonitor installation for the operator")
+		return nil
+	}
+	log.Info("exposing metric for installed knative version")
+	knativeVersion.WithLabelValues(version.Version, namespace).Set(float64(time.Now().Unix()))
```

Comment:
This namespace actually shouldn't be the operator's namespace, should it? I'd move this line to the beginning of the reconcile loop actually. It's also not openshift specific so let's move it into the general path?

Reply:
Then which namespace?
I feel we can remove `namespace` also because by default metric add `exported_namespace` as show in the screenshot.

Reply:
yeah I think it's fine either way. It'd be premature optimization for a maybe future world where you can have more than one knativeServing per cluster and have different versions for them... but that's faaaar out and we can always change later.

Reply:
To make it general planning to add changes to expose metric once after 
`log.Info("All deployments are available")`

Reply:
I'd actually expose it in any case. Healthyness stats should be written separately.